### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by 5m4r7**
create

### CODE REVIEW

**Comment by koko1928** [comment]
README.md will be better if you write usage instructions and libraries instead of just explaining the code.

**Review by koko1928 — APPROVED** [review]
It is better to write in more detail

**Comment by koko1928** [comment]
> README.md will be better if you write usage instructions and libraries instead of just explaining the code.

In this case, write where you found this site and how you collected it.

**Review by koko1928 — COMMENTED** [review]
Where did you find this site?
How to collect it?

**Comment by 5m4r7** [comment]
this appears to be a fraudulent site pretending to be a site that sells LINE stamps. I found and collected this site.

File: README.md
Comment (koko1928) [inline README.md:1]: This appears to be a fraudulent site pretending to be a site that sells LINE stamps. I found and collected this site.

**Review by koko1928 — CHANGES_REQUESTED** [review]

File: README.md
Comment (5m4r7) [reply README.md:1]: thank you

**Pull request merged**